### PR TITLE
Show ratings on cards

### PR DESCRIPTION
**Pull request by adgad**
/cc @matthew-andrews

### CODE REVIEW

File: server/models/article-v1.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ module.exports = function(article, opts) {
 		output.showMedia = true;
 	}
 
+	if(article.nextHackRating) {
```
Comment (commuterjoy) [inline server/models/article-v1.js]: hacks should have expiry dates!

Comment (matthew-andrews) [inline server/models/article-v1.js]: this would be hard to expire :grin:

Comment (matthew-andrews) [inline server/models/article-v1.js]: ah we could have a flag for the rating ^^

**Comment by matthew-andrews** [comment]
awesomes

File: server/app.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ var app = module.exports = express({
 
 app.get('/special-reports.?:extension(rss)?', require('./controllers/special-reports'));
 app.get('/news-feed.?:extension(rss)?', require('./controllers/news-feed'));
+app.get('/top-rated/:thingId', require('./controllers/top-reviews'));
```
Comment (matthew-andrews) [inline server/app.js]: might want to add the taxonomy in here too…… `/:thingType/:thingId` so we can do, e.g. top rated in London

http://next.ft.com/streams/regionsId/TnN0ZWluX0dMX0dCX0VOR19Mb25kb24%3D-R0w%3D

**Pull request merged**